tools/common/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/Azure/azure-sdk-for-go/tools/common
```

Comment:
Well... I am using `common` instead of `internal` to make sure that some external tools will not have to rewrite everything again. But since we will be bringing the release tool in our tools, I believe we could change this to internal...

Reply:
I have changed `common` to `internal`